core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/ShardedHttpExecutionServiceCluster.scala

```diff
@@ -148,6 +148,29 @@ class ShardedHttpExecutionServiceCluster (readMembers: Set[ClusterMember], submi
     }
   }
 
+  def callLevelMetadataForCostCalculation(submissionId: String, workflowId: String, execId: Option[ExecutionServiceId], userInfo: UserInfo): Future[JsObject] = {
+    val metadataParams = MetadataParams(Set(
+      "id",
+      "start",
+      "end",
+      "labels",
+      "executionEvents",
+      "runtimeAttributes",
+      "jobId",
+      "preemptible",
+      "callCaching:hit",
+      "jes",
+      "papi2",
+      "executionStatus",
+      "backend"
+    ), Set.empty, true)
+
```

Comment:
I think this needs "attempt" for each call as well (the number of times we've attempted to run)

Reply:
Same for the associated Hamm PR